apps/api/README.md

```
@@ -0,0 +1,21 @@
+## APIについてまとめときます
+
+cashier/sessions/セッションのID/products
+レジ画面の商品一覧取得API
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)